cip.go

```diff
@@ -337,7 +337,12 @@ func main() {
 
 	// Check the pull request
 	if *dryRunPtr {
-		err = sc.RunChecks(promotionEdges, []reg.PreCheck{})
+		err = sc.RunChecks(promotionEdges, []reg.PreCheck{
+			reg.ImageRemovalCheck{
+				*auditManifestRepoUrlPtr,
+				os.Getenv("PULL_BASE_SHA"),
```

Comment:
You should move out the call to os.Getenv() out of this argument list. As a general rule, unless you know for a fact that the function cannot fail, always execute such dependencies early and then pass along the results as an argument to another function call.